x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_keys_grid_page.tsx

```diff
@@ -388,14 +388,16 @@ export class APIKeysGridPage extends Component<Props, State> {
                 return {
                   value: username,
                   view: (
-                    <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
-                      <EuiFlexItem grow={false}>
-                        <EuiIcon type="user" />
-                      </EuiFlexItem>
-                      <EuiFlexItem grow={false}>
-                        <EuiText>{username}</EuiText>
-                      </EuiFlexItem>
-                    </EuiFlexGroup>
+                    <EuiToolTip delay="long" position="left" content={username}>
+                      <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+                        <EuiFlexItem grow={false}>
+                          <EuiIcon type="user" />
+                        </EuiFlexItem>
+                        <EuiFlexItem grow={false}>
+                          <EuiText>{username}</EuiText>
+                        </EuiFlexItem>
```

Comment:
> I experimented with trying to truncate the text with an ellipsis (using the text-overflow: ellipsis CSS style), but that doesn't seem to work with the flex layout, so I opted not to spend any more time on that.

Have you tried something like this (taken from [here](https://css-tricks.com/flexbox-truncated-text/))?
```suggestion
                        <EuiFlexItem
                          css={css`
                            min-width: 0;
                          `}
                          grow={false}
                        >
                          <EuiText className="eui-textTruncate">{username}</EuiText>
                        </EuiFlexItem>
```